internal/indexer/package_declaration.go

```diff
@@ -0,0 +1,63 @@
+package indexer
+
+import (
+	"go/token"
+	"go/types"
+
+	"golang.org/x/tools/go/packages"
+)
+
+// TODO: Handle testing packages differently.
+
+type ObjectLike interface {
```

Comment:
Why was it necessary to define this over `types.Object`?

Reply:
We have to make our own type, PkgDeclaration. I will add explanation above the struct